docs/content/about-iota/iota-wallet-dashboard/how-to/migration.mdx

```diff
@@ -0,0 +1,47 @@
+---
+description: Learn how to migrate your Stardust NFT and native assets using the IOTA Wallet Dashboard.
+teams:
+   - iotaledger/tooling
+tags: [iota-wallet-dashboard]
+---
+
+# Migrate Stardust Assets
+
+You can use the IOTA Wallet Dashboard to migrate Stardust NFTs,
+native assets that need migrating. 
+
+:::danger 
```

Comment:
The user cant really decide on not migrating some assets, so this danger can be removed imo